processor/structs.go

```diff
@@ -48,6 +50,7 @@ type FileJobCallback interface {
 // FileJob is a struct used to hold all of the results of processing internally before sent to the formatter
 type FileJob struct {
 	Language           string
+	Languages          []string // Used to hold potentially more than one language which populates language when determined
```

Comment:
I'm not a fan of having `Language` and `Languages` in the same struct. I think we should differentiate the names if they're both required - `PossibleLanguages` and `ActualLanguage` or something.

Reply:
`Language` will need to remain. This is because it keeps backwards compatibility with those calling direct into CountStats, of which a few applications do now.

Perhaps `PossibleLanguages` over `Languages` then. I'll make that change now.